lib/AST/Decl.cpp

```diff
@@ -2550,6 +2550,34 @@ bool ValueDecl::isUsableFromInline() const {
   return false;
 }
 
+/// Returns \c true if this declaration is *not* intended to be used directly
+/// by application developers despite of the visibility.
```

Comment:
We don't usually copy doc comments to the definition since then we have two places to keep up to date separately.